Sources/eppo/Constants.swift

```diff
@@ -1,5 +1,5 @@
 // todo: make this a build argument (FF-1944)
 public let sdkName = "ios"
-public let sdkVersion = "5.3.0-rc.1"
+public let sdkVersion = "5.3.0-rc.2"
```

Comment:
We'll need to decide which initialization options to keep and/or remove before doing a non-candidate release
* `withPersistentCache`
* `debugCallback`